src/CreeDictionary/CreeDictionary/paradigm/panes.py

```diff
@@ -641,6 +640,30 @@ class ColumnLabel(BaseLabelCell):
     prefix = "|"
 
 
+def parse_label(text: str, *, prefix: str) -> list[str]:
+    """
+    Parses labels in the common label syntax.
+
+    >>> parse_label("| Ind", prefix="|")
+    ['Ind']
+    >>> parse_label("# Fut # Def", prefix="#")
+    ['Fut', 'Def']
+    >>> parse_label("_ 1Sg _ 3Sg _ Obv", prefix="_")
+    ['1Sg', '3Sg', 'Obv']
+    """
+    splits = re.split(r" +", text)
+    if len(splits) % 2 != 0:
+        raise ParseError(f"Uneven number of space-separated segments in {text!r}")
+
+    tags = []
+    for actual_prefix, tag in pairs(splits):
+        if actual_prefix != prefix:
+            raise ParseError(f"Expected prefix {prefix!r} but saw {actual_prefix!r}")
```

Comment:
some minor optional golfing:

```suggestion
            raise ParseError(f"Expected {prefix=!r} but saw {actual_prefix!r}")
```